db/migrate/20150901141746_move_client_admin_role.rb

```diff
@@ -0,0 +1,11 @@
+class MoveClientAdminRole < ActiveRecord::Migration
+  def change
+    role = Role.find_or_create_by(name: 'client_admin')
+
+    User.client_admin_emails.each do |email_address|
```

Comment:
We should avoid using the `.client_admin_emails` method here, since it will presumably be removed and thus break this migration in the future.